docker/haproxy/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you don't use the `docker` folder, just delete it, we will not have `haproxy` for sure.
I would keep as simple as possible this repo, to avoid "noise"